code/edge_detection.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>At one point we should figure out a good metric to judge the "goodness" of an edge detection. For now our best judgement should be fine, but ideally the cirrhosis detection in the last step should tell us more about it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
definitely, we should also make sure to test on a bunch of images. Right now our solution might only work on this specific image.